### ISSUE TITLE
Add resolution hints for symbols available in the workspace

### ISSUE THREAD

**Issue by nfx** [microsoft/vscode-jupyter#5064]
In order to simplify development experience, i'd like to see so much used import hints in VSCode.
![image](https://user-images.githubusercontent.com/259697/110245724-bbd9c380-7f64-11eb-9c7a-8c4b9cc6fd7d.png)

**Comment by DavidKutu** [comment]
Thanks for the suggestion @nfx.

Could you describe your issue a bit deeper? Is Intellisense not giving you an option to import `UNPROCESSED` from the right place in your workspace?

**Comment by nfx** [comment]
@DavidKutu nope, intellisense does not do that :( how to reproduce - create a Pipenv project, add `click` as dependency, `pipenv sync`, wait till everything is installed, create a python file, type `print(UNPROCESSED)`, see red line highlight. And i'd expect a hint, like in IntelliJ (community edition + python plugin), to show me import suggestions.

**Comment by nfx** [comment]
@DavidKutu this one was in a python file. but i'd want to see it in a notebook as well.

i'm using insiders build (it broke last night!)

```
Extension version: 2021.3.600686576
VS Code version: Code - Insiders 1.54.0-insider (Universal) (e590188f17162393f50feec19263398e6fe02d13, 2021-02-26T20:39:12.763Z)
OS version: Darwin x64 19.6.0

A/B Experiments
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30263608
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30244315
pythonvsdeb440:30224570
pythonvsded773:30223139
pythonvspyt875:30259475
pythonvspyt639:30264824
pythontb:30258533
```
